CefSharp.Wpf/Experimental/ChromiumWebBrowserWithTouchSupport.cs

```diff
@@ -118,38 +181,53 @@ private void OnTouch(TouchEventArgs e)
             }
         }
 
+        //Both touch and stylus pen will Run OnStylusXXX.
```

Comment:
Please remove the random comments and add a remarks section to the class xml doc, provide a detailed summary of the different cases please. The details are spread out and that's not ideal

Reply:
Mouse and stylus events won't present in the one event please.
But one stylus operation will raise both mouse event and stylus event.